includes/class-wcpdf-main.php

```diff
@@ -176,6 +176,13 @@ public function attach_pdf_to_email ( $attachments, $email_id, $order, $email =
 				$attachments[] = $pdf_path;
 
 				do_action( 'wpo_wcpdf_email_attachment', $pdf_path, $document_type, $document );
+
+				// log document creation to order notes
+				$debug_settings = get_option( 'wpo_wcpdf_settings_debug', array() );
+				if( isset( $debug_settings['log_to_order_notes'] ) ) {
+					$action = __( 'attachment', 'woocommerce-pdf-invoices-packing-slips' );
```

Comment:
this should be a code/slug (and I think `$context` is a better description), not a translatable string.